src/grasp_generator.cpp

```diff
@@ -721,58 +720,137 @@ bool GraspGenerator::addGrasp(const Eigen::Affine3d& grasp_pose, const GraspData
   // origin on palm, z pointing outward, x perp to gripper close, y parallel to gripper close direction
   // Transform the grasp pose
 
-  Eigen::Affine3d eef_pose = grasp_pose * grasp_data->grasp_pose_to_eef_pose_;
+  // NOTE: This is the grasp pose NOT the end effector pose!
```

Comment:
Then why not just use grasp_pose?

Reply:
grasp_pose and eef_pose are used interchangeably at different places in the moveit grasps code base. This was an attempt to disambiguate them. I have updated the code to make it more clear.